app/src/androidTest/java/com/example/henzoshimada/feeltrip/FeelTripTests.java

```diff
@@ -67,12 +67,11 @@ public void testGetMoodTask(){
         assertEquals("test getMood", "user1", moods.get(0).getUser());
     }
 
-    // does not pass
     public void testEditTask() throws DescriptionTooLongException {
-        ArrayList<Mood> moods = new ArrayList<>();
-        ElasticSearchController.GetMoodTask getMoodTask = new ElasticSearchController.GetMoodTask("user");
+        ArrayList<Mood> moods = new ArrayList<Mood>();
+        ElasticSearchController.GetMoodTask getMoodTask = new ElasticSearchController.GetMoodTask();
```

Comment:
It's just a test case, it allowed me to grab all that was in our database as opposed to only "user1" posts